### ISSUE TITLE
something went wrong when updating homebridge

### ISSUE THREAD

**Issue by tvillingett** [nfarina/homebridge#549]
It cannot complete, and when I restarted homebridge (on a RPi) it was missing.
I tried to reinstall it using npm install -g homebridge, but get this (I cut the important part I hope):
Anyone that have seen this?
```
1010 silly cache afterAdd ed25519@0.0.3
1011 verbose afterAdd /root/.npm/ed25519/0.0.3/package/package.json not in flight; writing
1012 verbose afterAdd /root/.npm/ed25519/0.0.3/package/package.json written
1013 verbose about to build /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs
1014 verbose unlock done using /root/.npm/_locks/hap-nodejs-ad226527e82daf47.lock for /usr/local/lib/node_modules/homebridge
/node_modules/hap-nodejs
1015 verbose about to build /usr/local/lib/node_modules/homebridge
1016 verbose unlock done using /root/.npm/_locks/homebridge-a5618ac2152da688.lock for /usr/local/lib/node_modules/homebridge
1017 verbose stack Error: Failed to parse json
1017 verbose stack No data, empty input at 1:1
1017 verbose stack
1017 verbose stack ^
1017 verbose stack     at parseError (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:379:11)
1017 verbose stack     at parseJson (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:68:23)
1017 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:48:5
1017 verbose stack     at FSReqWrap.readFileAfterClose [as oncomplete] (evalmachine.<anonymous>:380:3)
1018 verbose cwd /home/pi
1019 error Linux 4.1.13-v7+
1020 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "homebridge"
1021 error node v4.0.0
1022 error npm  v2.14.2
1023 error file /root/.npm/color-convert/1.0.0/package/package.json
1024 error code EJSONPARSE
1025 error Failed to parse json
1025 error No data, empty input at 1:1
1025 error
1025 error ^
1026 error File: /root/.npm/color-convert/1.0.0/package/package.json
1027 error Failed to parse package.json data.
1027 error package.json must be actual JSON, not just JavaScript.
1027 error
1027 error This is not a bug in npm.
1027 error Tell the package author to fix their package.json file. JSON.parse
1028 verbose exit [ 1, true ]
1029 verbose unbuild lib/node_modules/homebridge/node_modules/chalk/node_modules/ansi-styles
1030 info preuninstall ansi-styles@2.2.0
1031 info uninstall ansi-styles@2.2.0
1032 verbose unbuild rmStuff ansi-styles@2.2.0 from /usr/local/lib/node_modules
1033 verbose unbuild rmStuff in /usr/local/lib/node_modules/homebridge/node_modules/chalk/node_modules
1034 info postuninstall ansi-styles@2.2.0
1035 silly gentlyRm /usr/local/lib/node_modules/homebridge/node_modules/chalk/node_modules/ansi-styles is being purged from 
base /usr/local
1036 verbose gentlyRm don't care about contents; nuking /usr/local/lib/node_modules/homebridge/node_modules/chalk/node_modul
es/ansi-styles
1037 silly vacuum-fs purging /usr/local/lib/node_modules/homebridge/node_modules/chalk/node_modules/ansi-styles
1038 silly vacuum-fs quitting because other entries in /usr/local/lib/node_modules/homebridge/node_modules/chalk/node_module
s
1039 verbose unbuild lib/node_modules/homebridge/node_modules/chalk
1040 info preuninstall chalk@1.1.1
1041 info uninstall chalk@1.1.1
1042 verbose unbuild rmStuff chalk@1.1.1 from /usr/local/lib/node_modules
1043 verbose unbuild rmStuff in /usr/local/lib/node_modules/homebridge/node_modules
1044 info postuninstall chalk@1.1.1
1045 silly gentlyRm /usr/local/lib/node_modules/homebridge/node_modules/chalk is being purged from base /usr/local
1046 verbose gentlyRm don't care about contents; nuking /usr/local/lib/node_modules/homebridge/node_modules/chalk
1047 silly vacuum-fs purging /usr/local/lib/node_modules/homebridge/node_modules/chalk
1048 silly vacuum-fs quitting because other entries in /usr/local/lib/node_modules/homebridge/node_modules
1049 verbose unbuild lib/node_modules/homebridge/node_modules/hap-nodejs
1050 info preuninstall hap-nodejs@0.2.5
1051 info uninstall hap-nodejs@0.2.5
1052 verbose unbuild rmStuff hap-nodejs@0.2.5 from /usr/local/lib/node_modules
1053 verbose unbuild rmStuff in /usr/local/lib/node_modules/homebridge/node_modules
1054 info postuninstall hap-nodejs@0.2.5
1055 silly gentlyRm /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs is being purged from base /usr/local
1056 verbose gentlyRm don't care about contents; nuking /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs
1057 silly vacuum-fs purging /usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs
1058 silly vacuum-fs quitting because other entries in /usr/local/lib/node_modules/homebridge/node_modules
1059 verbose unbuild lib/node_modules/homebridge
1060 info preuninstall homebridge@0.2.19
1061 info uninstall homebridge@0.2.19
1062 verbose unbuild rmStuff homebridge@0.2.19 from /usr/local/lib/node_modules
1063 silly gentlyRm /usr/local/bin/homebridge is being gently removed
1064 silly gentlyRm verifying /usr/local is an npm working directory
1065 silly gentlyRm containing path /usr/local is under npm's control, in /usr/local
1066 silly gentlyRm deletion target /usr/local/bin/homebridge is under /usr/local
1067 verbose gentlyRm vacuuming from /usr/local/bin/homebridge up to /usr/local
1068 info postuninstall homebridge@0.2.19
1069 silly gentlyRm /usr/local/lib/node_modules/homebridge is being purged from base /usr/local
1070 verbose gentlyRm don't care about contents; nuking /usr/local/lib/node_modules/homebridge
1071 silly vacuum-fs purging /usr/local/lib/node_modules/homebridge
1072 silly vacuum-fs quitting because other entries in /usr/local/lib/node_modules
```

**Comment by RecognitionDesigns** [comment]
I had the same issue!

I tried to update Homebridge to the version that supports iOS9.3 public beta but i missed off the word 'homebridge' at the end.

When I tried to restart Homebridge i had the same issue as you, Homebridge could not be found.
Tried reinstalling and was getting the same issue as you.

Only way I got Homebridge working was to totally wipe me Rpi and start from scratch. (I only use it for Homebridge so not much lost doing this for me), but then my iPhone couldn't add the Homebridge. (iPhone was still on iOS 9.2.1)
Updated to the iOS 9.3 public beta and its working again!
